web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/AgentWaitingDialog.tsx

```diff
@@ -0,0 +1,147 @@
+import { usePingTeleport } from 'teleport/Discover/Shared/PingTeleportContext';
+import { Mark, TextIcon, useShowHint } from 'teleport/Discover/Shared';
+import { HintBox } from 'teleport/Discover/Shared/HintBox';
+
+type AgentWaitingDialogProps = {
+  joinToken: JoinToken;
+  status: string;
+  clusterName: string;
+  setWaitingResult(cluster: Kube): void;
+  close(): void;
+};
+
+export function AgentWaitingDialog({
+  joinToken,
+  status,
+  clusterName,
+  setWaitingResult,
+  close,
+}: AgentWaitingDialogProps) {
+  const { result, active } = usePingTeleport<Kube>(joinToken);
+  setWaitingResult(result);
+  const showHint = useShowHint(active);
+
+  function hintMessage() {
+    if (showHint && !result) {
+      return (
+        <Box mb={3}>
+          <HintBox header="We're still looking for your server">
+            <Text mb={3}>
+              There are a few of possible reasons for why we haven't been able
+              to detect your Kubernetes cluster.
+            </Text>
+
+            <Text mb={1}>- The cluster doesn't have active nodes.</Text>
+
+            <Text mb={1}>
+              - The manual command was not run on the server you were trying to
+              add.
+            </Text>
+
+            <Text mb={3}>
+              - The Teleport Service could not join this Teleport cluster. Check
+              the logs for errors by running
+              <br />
+              <Mark>kubectl logs -l app=teleport-agent -n teleport-agent</Mark>
+            </Text>
+
+            <Text>
+              We'll continue to look for your Kubernetes cluster whilst you
+              diagnose the issue.
+            </Text>
+          </HintBox>
+        </Box>
+      );
+    }
+  }
+
+  function content() {
+    if (status === 'awaitingAgent') {
+      return (
+        <>
+          <Text bold caps mb={4}>
+            EKS Cluster Enrollment
+          </Text>
+          <AnimatedProgressBar mb={3} />
+          <TextIcon
+            mb={3}
+            css={`
+              white-space: pre;
+            `}
+          >
+            <Icons.Check size="medium" />
+            <Text>1. Installing Teleport agent</Text>
+          </TextIcon>
+          <TextIcon
+            mb={3}
+            css={`
+              white-space: pre;
+            `}
+          >
+            <Icons.Clock size="medium" />
+            <Text>
+              2. Waiting for the Teleport agent to come online (1-5 minutes)...
+            </Text>
+          </TextIcon>
+          {hintMessage()}
+          <ButtonPrimary width="100%" onClick={close}>
+            Cancel
+          </ButtonPrimary>
+        </>
+      );
+    } else {
+      return (
+        <>
+          <Text bold caps mb={4}>
+            EKS Cluster Enrollment
+          </Text>
+          <Text mb={3} style={{ display: 'flex' }}>
+            <Icons.Check size="small" ml={1} mr={2} color="success" />
+            Cluster "{clusterName}" was successfully enrolled.
+          </Text>
```

Comment:
color for `success` recently changed to `success.main` which is why color wasn't showing up

```suggestion
          <Flex mb={3}>
            <Icons.Check size="small" ml={1} mr={2} color="success.main" />
            Cluster "{clusterName}" was successfully enrolled.
          </Flex>
```